### PR TITLE
Rev 1260: Separate Request and Response models into new package

### PR DESCRIPTION
**Pull request by whwatkinson**
Data models (and their immediate dependencies) should be packaged separately. Several upstream packages have a dependency on projectneutron, which brings nearly 1gb of imports.

The only heavyweight import for the data models is pydantic - at ~50MB.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "Operating System :: OS Independent",
+    ],
+    package_dir={"api-models": ""},
+    packages=setuptools.find_packages(),
+    include_package_data=True,
+    python_requires=">=3.9",
+    install_requires=[
+        "flake8==3.9.2",
```
Comment (fadesibert) [inline setup.py:22]: flake8 is only needed for development, not for package itself

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "geojson == 2.5.0",
+        "phonenumbers == 8.12.13",
+        "pydantic == 1.8.2",
+        "pytest == 6.2.4",
```
Comment (fadesibert) [inline setup.py:26]: ditto pytest

**Review by fadesibert — DISMISSED** [review]

**Review by JammyL — DISMISSED** [review]

File: api_models/response_models/address.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from typing import Optional
+
+# from geojson_pydantic.features import FeatureCollection
+# from geojson_pydantic.geometries import Point
```
Comment (JammyL) [inline api_models/response_models/address.py:4]: commented out code should go

**Review by JammyL — APPROVED** [review]

**Comment by whwatkinson** [comment]
commit master not sign, rebase failed.

Opened rev-1260b to continue

**Pull request closed without merging**